assets/js/hooks/useHasScrolledEffect.test.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * `useHasScrolled` hook tests.
```

Comment:
Please update references to use the updated name.

Reply:
👍